src/pages/VerifyEmail.tsx

```diff
@@ -1,5 +1,9 @@
 import { useEffect, useMemo, useState } from "react";
 import { useLocation } from "react-router-dom";
+import Background from "../assets/Background.jpg";
+import { Navbar } from "../components";
```

Comment:
Add the `ErrorComponent` and `SuccessComponent` to line 4, so that it looks like `import { Navbar, ErrorComponent, SuccessComponent } from "../components";`

Reply:
sure thing :)